### PR TITLE
Add info about another attack type

### PR DESCRIPTION
**Pull request by kenchris**
***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/compute-pressure/pull/251.html" title="Last updated on Mar 6, 2024, 10:42 AM UTC (4288ec5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/compute-pressure/251/b7a2bbe...kenchris:4288ec5.html" title="Last updated on Mar 6, 2024, 10:42 AM UTC (4288ec5)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1353,7 +1353,48 @@ <h4>Cross-site covert channel</h4>
       a video conferencing session will typically exert sustained pressure on the CPU that makes it harder to
       manipulate the pressure state in a predictive manner.
     </div>
-
+    <h4>Targeted deanonymization attacks</h4>
```
Comment (anssiko) [inline index.html:1356]: s/deanonymization/de-anonymization/g

File: index.html [hunk 2]
```diff
@@ -1353,7 +1353,48 @@ <h4>Cross-site covert channel</h4>
...
+    <p>
+      Targeted deanonymization attacks constitute a critical class of threats that jeopardize a users anonymity.
```
Comment (anssiko) [inline index.html:1358]: user's

File: index.html [hunk 3]
```diff
@@ -1353,7 +1353,48 @@ <h4>Cross-site covert channel</h4>
...
+      understand if a target of a cross-origin navigation (e.g. an iframe or pop-up window from another site)
+      performed a CPU-intensive operation.
+    </p>
+    <p>
+      Techniques such as <a href="https://en.wikipedia.org/wiki/Pop-up_ad#Pop-under_ads">pop-under</a> and
+      <a href="https://www.usenix.org/system/files/sec22-zaheri.pdf">tab-under</a> can be used to hide the loading
+      from the user.
+    </p>
+    <p>
+      A practical attack is that the malicious website could open e.g., a popup to a resource on a victim site
```
Comment (anssiko) [inline index.html:1385]: One possible attack ... website opens ...

File: index.html [hunk 4]
```diff
@@ -1353,7 +1353,48 @@ <h4>Cross-site covert channel</h4>
...
+      to which the user is logged in (e.g. a video streaming site or online document editor) pointing to a
+      resource shared with specific users.
+    </p>
+    <p>
+      Assuming that loading the resource puts increased pressure on the CPU, this would create a side-channel
+      reveals to the attacking site if the user is logged into an account with access to the resource,
+      deanonymizing the user.
+    </p>
+    <p>
+      Given that modern CPUs recover quickly from high pressure, a mitigation strategy could be to temporarily
```
Comment (anssiko) [inline index.html:1395]: one possible mitigation strategy is to

**Review by anssiko — APPROVED** [review]
LGTM with some comments.

File: index.html [hunk 5]
```diff
@@ -1353,7 +1353,48 @@ <h4>Cross-site covert channel</h4>
...
+    <h4>Targeted deanonymization attacks</h4>
+    <p>
+      Targeted deanonymization attacks constitute a critical class of threats that jeopardize a users anonymity.
+      These attacks allow a malicious or partially compromised website (referred to as the “malicious site”) to
+      ascertain whether a website visitor possesses a specific public identifier, such as an email address or a
+      social media handle.
+    </p>
+    <p>
+      While anonymity may be a luxury for some, for certain individuals, it is far more than that—it is a matter
+      of survival. Consider for instance those who engage in political protests, work as journalists covering
```
Comment (arskama) [inline index.html:1365]: nit
"Consider for instance those who engage in political protests, work as journalists covering sensitive topics, etc"
maybe:
"Consider for instance those who engage with freedom of information."

Comment (kenchris) [reply index.html:1365]: I think that is a bit weak and harder to understand

**Comment by arskama** [comment]
LGTM with suggestion.

File: index.html (hunk 5 above)
Comment (arskama) [reply index.html:1365]: it s just more generic

**Pull request merged**